tests/xharness/Xharness.Tests/Hardware/Tests/DevicesTest.cs

```diff
@@ -72,7 +72,7 @@ public void LoadAsyncProcessErrorTest (bool timeout)
 
 			MlaunchArgument sdkRootArg = passedArguments.Where (a => a is SdkRootArgument).FirstOrDefault();
 			Assert.IsNotNull (sdkRootArg, "sdk arg missing");
-			AssertArgumentValue (sdkRootArg, sdkPath, "sdk arg wrong");
+			AssertArgumentValue (sdkRootArg, $"--sdkroot {sdkPath}", "sdk arg wrong");
```

Comment:
is this related to the change ?

Reply:
No, but the tests failed, there was a commit that should have updated this.

Reply:
That should be a separate PR or, minimally, mentioning in the commit message.
Also it should link to the original commit and check why this was merged if not working.

Reply:
@spouliot issue with these tests, they are not in our CI, xharness is not running its own tests, might be a good idea to add that.

Reply:
Tests that don't run don't exists :(
Let's move that unrelated fix into a PR that enable the tests :)

Reply:
I opened a PR to fix these and couple of others that are failing: https://github.com/xamarin/xamarin-macios/pull/8208

Reply:
@premun thx!!!! I'll revert that change then!